doc/Metrics.md

```diff
@@ -62,12 +64,65 @@ added to the deployment with the following parameters:
   "version": "1.0"
 }
 ```
-After the modules are deployed, Chronograf can be reached at localhost:8888.
+After the modules are deployed, Chronograf can be reached at http://localhost:8888. InfluxDb should be configured as a data source for Chronograf. The InfluxDb instance should be reachable at http://influxdb:8086.
```

Comment:
Is the port binding to a host port in the JSON above required?

Reply:
Yes so that you can point to it from a browser. Why, do you think its not?

Reply:
Port mapping opens up an endpoint on the host. So unless we expect downstream devices to directly access the Influx DB service we should perhaps default to not port-mapping it.

Will the user be pointing the browser at the chorongraf service or the Influx DB service?

Reply:
Chronograf needs to reach Influx at that port.

Reply:
Since chronograf is just another container on the same bridged network, it can directly connect to the InfluxDB container without the need for port mapping InfluxDB to the host.